tests/Tests.elm

```diff
@@ -85,6 +87,19 @@ regressions =
             \positiveInt ->
                 positiveInt
                     |> Expect.greaterThan 0
+        , fuzz
+            (custom (Random.int 1 8) Shrink.noShrink)
```

Comment:
I'd like a comment here explaining why `intRange` isn't used, and how this differs. Or rather, I'd like to use `intRange` instead and note that the edges are weighted a bit higher.